lmp/build.sh

```diff
@@ -133,7 +133,7 @@ if [ -d "${archive}" ] ; then
 	# Handle user provided extra artifacts
 	if [ ! -z "${EXTRA_ARTIFACTS}" ]; then
 		for extra_file in ${EXTRA_ARTIFACTS}; do
-			cp ${DEPLOY_DIR_IMAGE}/${extra_file} ${archive}/
+			cp ${DEPLOY_DIR_IMAGE}/${extra_file} ${archive}/ || true
```

Comment:
Maybe we should do something like:
~~~
if [ -f ${DEPLOY_DIR_IMAGE}/${extra_file} ] ; then
    cp ${DEPLOY_DIR_IMAGE}/${extra_file} ${archive}/
else
  echo "Skipping archiving ${extra_file} - file does not exist"
fi
~~~

Reply:
Yeah, probably best to log a message since the user is explicitly asking for this file. I'll update the PR.